install/kubernetes/helm/istio/charts/galley/templates/validatingwebhookconfiguration.yaml.tpl

```diff
@@ -110,6 +110,9 @@ webhooks:
         - quotas
         - reportnothings
         - tracespans
+        - handlers
+        - instances
+        - zipkins
```

Comment:
templates and adapters seem to be missing here.

Reply:
Thanks for catching this.  Added; and manually tested with both valid and invalid examples.

Reply:
Can one of you add tests for these?

Reply:
@ozevren I want to add tests for these.  I want to add the tests as a follow-on after https://github.com/istio/istio/pull/12710 is merged because I don't want conflicts with _tests/integration/galley/validation/data.gen_test.go_ changes there.  I won't close https://github.com/istio/istio/issues/12886 until the planned test PR has been merged.

Reply:
/lgtm
Sounds good. Thanks